mep/books/templates/books/work_cardlist.html

```diff
@@ -0,0 +1,77 @@
+{% extends 'base.html' %}
+{% load render_bundle from webpack_loader %}
+{% load static mep_tags %}
+
+{% block page-subtitle %}{{ work.title }} Cards · Library Holdings · {% endblock %}
+
+{% block js %}
+{% render_bundle 'activities' 'js' %}
+{% endblock %}
+
+{% block header %}
+{% include 'snippets/header.html' with title=work.title style="detail" %}
+{% endblock %}
+
+{% block main-style %}tabbed white{% endblock %}
+
+{% block content %}
+{# breadcrumbs & tabs #}
+{% include 'snippets/breadcrumbs.html' %}
+<nav class="tabs" aria-label="tabs">
+    <li class="tab">
+        <a href="{% url 'books:book-detail' slug=work.slug %}">bibliography</a>
+    </li>
+    <li class="tab">
+        <a href="{% url 'books:book-circ' slug=work.slug %}">circulation</a>
+    </li>
+    <li class="tab" aria-selected="true">
+        <a href="{% url 'books:book-card-list' slug=work.slug %}">cards</a>
+    </li>
+</nav>
+
+<section class="book-cards">
+    {% if footnotes %}
+    <ol class="card-results">
```

Comment:
can this benefit from using the footnotes `card_results.html` snippet at all? seems to be some overlap

Reply:
It's different enough that it was hard to see how to reuse them:
- in this case, we're iterating on footnotes with images instead of the images directly
- we want to display the date for the particular event associated with this work, not for the card as a whole
- we want to link to the card image detail page with the specific event highlighted